### PR TITLE
Update nvALT.app to support macOS Sierra

### PR DESCRIPTION
**Pull request by Ben-Chapman**
This is an update to the nvALT cask version 2.2120.

nvALT was updated to support macOS Sierra and this update is compatible
with versions of OS X >=10.9. The previous version os nvALT was
preserved for those users with OS X < 10.9.

### CODE REVIEW

File: Casks/nvalt.rb [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+  if MacOS.version <= :mavericks
+    version '2.2b111'
+    sha256 'd787ddf92730bb03ba084e72bc6fb5f4fbd42731fa3531476af9eb3ce39e1cd0'
+    # abyss.designheresy.com/nvaltb was verified as official when first introduced to the cask
+    url "http://abyss.designheresy.com/nvaltb/nvalt#{version}.zip"
+  else
+    version '2.2b120'
+    sha256 'e3ebdc012d5bba6e0e46173daa7c7548eca947958355b37d6ae6040e85b79153'
+    # abyss.designheresy.com/ was verified as official when first introduced to the cask
+    url "http://abyss.designheresy.com/nvALT#{version.sub(%r{b}, '')}.zip"
```
Comment (vitorgalvao) [inline Casks/nvalt.rb]: `#{version.delete('b')}`

File: Casks/nvalt.rb [hunk 2]
```diff
@@ -1,11 +1,18 @@
...
-  sha256 'd787ddf92730bb03ba084e72bc6fb5f4fbd42731fa3531476af9eb3ce39e1cd0'
+  if MacOS.version <= :mavericks
+    version '2.2b111'
+    sha256 'd787ddf92730bb03ba084e72bc6fb5f4fbd42731fa3531476af9eb3ce39e1cd0'
+    # abyss.designheresy.com/nvaltb was verified as official when first introduced to the cask
+    url "http://abyss.designheresy.com/nvaltb/nvalt#{version}.zip"
+  else
+    version '2.2b120'
+    sha256 'e3ebdc012d5bba6e0e46173daa7c7548eca947958355b37d6ae6040e85b79153'
+    # abyss.designheresy.com/ was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/nvalt.rb]: Remove trailing slash.

File: Casks/nvalt.rb [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+    url "http://abyss.designheresy.com/nvALT#{version.sub(%r{b}, '')}.zip"
+  end
 
-  # abyss.designheresy.com/nvaltb was verified as official when first introduced to the cask
-  url "http://abyss.designheresy.com/nvaltb/nvalt#{version}.zip"
   appcast 'http://abyss.designheresy.com/nvalt2/nvalt2main.xml',
```
Comment (vitorgalvao) [inline Casks/nvalt.rb]: This needs to be moved inside the `else` block.

Also, make it `http://abyss.designheresy.com/nvalt#{version.major}/nvalt#{version.major}main.xml`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/nvalt.rb (hunk 1 above)
Comment (Ben-Chapman) [inline Casks/nvalt.rb]: Ah, yes. That's way better.

File: Casks/nvalt.rb (hunk 3 above)
Comment (Ben-Chapman) [inline Casks/nvalt.rb]: Good catch

**Comment by Ben-Chapman** [comment]
@vitorgalvao the requested changes have been made. Thanks for the review.

**Review by vitorgalvao — APPROVED** [review]

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**